webapp/src/ts/modals/mm-modal/mm-modal.ts

```diff
@@ -26,7 +30,15 @@ export class MmModal {
   @Input() disableSubmit;
   @Input() danger;
 
-  constructor() {
+  constructor(private store: Store) {
+  }
+
+  ngOnInit() {
+    this.store
+      .select(Selectors.getMinimalTabs)
+      .subscribe(minimalTabs => {
```

Comment:
Because mm-modal is a generic approach I'll take the general approach to subscribe and then unsubscribe on destroy, despite now is only useful for the tour modal.

Reply:
I think we should really not pollute the mm-modal abstract. If possible, please subscribe in the child modal